src/Core/Commander/Interfaces/ApiInterface/ApiGlobe.js

```diff
@@ -668,7 +664,8 @@ ApiGlobe.prototype.setSceneLoaded = function setSceneLoaded() {
 ApiGlobe.prototype.setCenter = function setCenter(coordinates, isAnimated) {
     isAnimated = isAnimated || this.isAnimationEnabled();
     eventCenter.oldCenter = this.getCenter();
-    const position3D = this.scene.getEllipsoid().cartographicToCartesian(new GeoCoordinate(coordinates.longitude, coordinates.latitude, 0, UNIT.DEGREE));
+    const position3D = new Coordinates('EPSG:4326', coordinates.longitude, coordinates.latitude, 0)
+        .as('EPSG:4328').xyz();
```

Comment:
[4328](http://spatialreference.org/ref/epsg/4328/) is deprecated use [4978](http://spatialreference.org/ref/epsg/4978/)

what returns `position3D.longitude(DEGREE) `?